plugins/services/src/js/structs/ServiceTree.js

```diff
@@ -388,6 +388,10 @@ module.exports = class ServiceTree extends Tree {
 
   getTaskCount() {
     return this.flattenItems().getItems().reduce(function(memo, service) {
+      if (service instanceof ServiceTree) {
```

Comment:
`#flattenItems` should recursively flatten the tree. Did you run in to an error or are you being extra cautious? :)